packages/components/Checkbox/src/Checkbox.settings.ts

```diff
@@ -6,10 +6,11 @@ export const checkboxSelectActionLabel = 'Toggle the Checkbox';
 export const settings: IComposeSettings<ICheckboxType> = [
   {
     tokens: {
-      borderColor: 'menuItemText',
-      color: 'menuItemText',
-      backgroundColor: 'menuBackground',
+      backgroundColor: 'transparent',
       textBorderColor: 'transparent',
+      borderColor: 'checkboxBorderColor',
+      checkmarkColor: 'checkmarkColor',
+      checkmarkVisibility: 0,
```

Comment:
Are these new tokens?

Reply:
Yes these are the new tokens. Added them to respective theme as well.